test/benchmarks/pulse/schedule_construction.py

```diff
@@ -16,15 +16,16 @@
 # pylint: disable=attribute-defined-outside-init
 
 import numpy as np
-from qiskit.pulse import Schedule, Gaussian, DriveChannel, SamplePulse, Play
+
+from qiskit.pulse import Schedule, Gaussian, DriveChannel, Play, Waveform
 
 
 def build_sample_pulse_schedule(number_of_unique_pulses, number_of_channels):
     rng = np.random.RandomState(42)
     sched = Schedule()
     for _ in range(number_of_unique_pulses):
         for channel in range(number_of_channels):
-            sched.append(Play(SamplePulse(rng.random(50)),
+            sched.append(Play(Waveform(rng.random(50)),
                               DriveChannel(channel)))
```

Comment:
```suggestion
                              DriveChannel(channel)),
                              inplace=True)
```
Would also need to be added for other `insert`/`append` instructions. The current benchmarks are broken as the schedule is not actually growing since the returned `sched` is not used.